pkg/services/grafana-apiserver/service.go

```diff
@@ -233,9 +233,15 @@ func (s *service) start(ctx context.Context) error {
 	o := options.NewRecommendedOptions("/registry/grafana.app", Codecs.LegacyCodec(groupVersions...))
 	o.SecureServing.BindAddress = s.config.ip
 	o.SecureServing.BindPort = s.config.port
+
 	o.Authentication.RemoteKubeConfigFileOptional = true
 	o.Authorization.RemoteKubeConfigFileOptional = true
 
+	if s.config.aggregationEnabled {
+		o.Authentication.SkipInClusterLookup = true
+		o.Authentication.RequestHeader.ClientCAFile = s.config.requestHeaderClientCAFile
```

Comment:
is it possible to have aggregationEnabled without a requestHeaderClientCAFile?

do we ever want/need SkipInClusterLookup = false? (IIUC, we are never running in a state where we have config maps https://github.com/kubernetes/apiserver/blob/master/pkg/server/options/authentication.go#L206)

Reply:
`requestHeaderClientCAFile` is necessary with `SkipInClusterLookup`. Without the skip, the leaf server will try to look up `extension-apiserver-authentication` configmap which will require a kubeconfig to the parent server.

Here, we are skipping the need to provide that kubeconfig by providing a static CA that Grafana can trust.